Monika After Story/game/script.rpy

```diff
@@ -28,6 +28,13 @@ label start:
     $ config.allow_skipping = True
 
     #Jump to the space room.
+    if persistent.autoload:
+        #Stop the title screen music
+        if persistent.current_track:
+            $ mas_startup_song()
+        else:
+            stop music
+        jump ch30_loop
```

Comment:
I would say its safer to jump to ch30_preloop instead, this makes sure that certain globals get set (like `closed_self` and others)